tests/sanity/sanity_test.go

```diff
@@ -142,6 +151,7 @@ func initCSIDriverForSanity(t *testing.T) *csiDriver.IBMCSIDriver {
 	// Create fake provider and mounter
 	provider, _ := NewFakeSanityCloudProvider("", logger)
 	mounter := mountManager.NewFakeSafeMounter()
+	//mounter := mountManager.NewSafeMounter()
```

Comment:
Why we have two same lines. One commented other not

Reply:
Removed the commented line. One was  for  `NewSafeMounter()` and other for  `NewFakeSafeMounter`